go.mod

```diff
@@ -15,8 +16,8 @@ require (
 	github.com/onsi/ginkgo v1.15.0 // indirect
 	github.com/onsi/gomega v1.10.5 // indirect
 	github.com/pborman/uuid v1.2.0
+	github.com/prometheus/client_golang v1.10.0
```

Comment:
I remember `prometheus` is monitoring tools.
Is used here code?
or auto importing this?

Reply:
Yeah. The Stargate supports the [Telemetry](https://docs.cosmos.network/master/core/telemetry.html) feature which collects some metrics to Prometheus, even though I haven't read it seriously.

`x/wasm` modules have some telemetry-related options: https://github.com/CosmWasm/wasmd/blob/d2e35c249e39e90c5a85e03e0b75800431612b20/cmd/wasmd/root.go#L205

Reply:
OK. I got it!